README.md

```diff
@@ -86,107 +86,104 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 ## 📋 Requirements
 
-- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora).
-  We anticipate support for more operating systems in the future.
+- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
+
+   📋  When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
+
 - C++ compiler
 - Python 3.10 or Python 3.11
-- Approximately 60GB disk space (entire process)
 
-> **NOTE:** Python 3.12 is currently not supported, because some dependencies don't work on Python 3.12, yet.
-<!-- -->
-> **NOTE:** When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
+   📋 Python 3.12 is currently not supported. Some InstructLab dependencies don't work on Python 3.12, yet.
 
-## ✅ Getting started
+- Approximately 60GB disk space (entire process)
```

Comment:
@cdoern is this still accurate do you think?

Reply:
no, this is way off, probably requires 500GB

Reply:
Yeah that seems like alot 😕